app/views/optOut/ConfirmedOptOutView.scala.html

```diff
@@ -0,0 +1,374 @@
+                    link = externalRefLink,
+                    messageKey = "optout.confirmedOptOut.reportQuarterly.desc3.anchor-text"
+                )
+
+                Html(messages("optout.confirmedOptOut.reportQuarterly.desc3", anchor.toString()))
+            }
+        }
+    </div>
+}
+
+@submitTaxReturnParagraph1 = {
+    @p(id = Some("now-you-have-opted-out")) {
+        @messages("optout.confirmedOptOut.submitTax.confirmed.p1")
+        @link(link = selfAssessmentTaxReturnLink, id = Some("self-assessment-tax-return-link"), messageKey = "optout.confirmedOptOut.submitTax.confirmed.p1.link", target = Some("_blank"), additionalOpenTabMessage = Some("."))
+    }
+}
+
+@submitTaxReturnParagraph2 = {
+    @p(id = Some("tax-year-reporting-quarterly")) {
+        @messages("optout.confirmedOptOut.submitTax.confirmed.p2")
+        @link(link = compatibleSoftwareLink, id = Some("compatible-software-link"), messageKey = "optout.confirmedOptOut.submitTax.confirmed.p2.link", target = Some("_blank"), additionalOpenTabMessage = Some("."))
+    }
+}
+
+@mainTemplate(
+    pageTitle = messages("optout.confirmedOptOut.heading"),
+    isAgent = isAgent,
+    btaNavPartial = user.btaNavPartial,
+) {
+
+    @if(confirmedOptOutViewScenarios == Scenario1Content) {
+        @greenPanelScenario1And4
+        @revisedDeadlinesBlock
+        <div id="submit-tax" class="govuk-!-margin-bottom-7  govuk-!-margin-top-8">
+            @h2(msg = "optout.confirmedOptOut.submitTax", optId = Some("submit-your-tax-return-heading"))
+            @submitTaxReturnParagraph1
+        </div>
+        @scenario1ReportingObligationsSection
+    }
+
+    @if(confirmedOptOutViewScenarios == Scenario2Content) {
+        @greenPanelScenario2
+        @revisedDeadlinesBlock
+        <div id="submit-tax" class="govuk-!-margin-bottom-7  govuk-!-margin-top-8">
+            @h2(msg = "optout.confirmedOptOut.submitTax", optId = Some("submit-your-tax-return-heading"))
+            @submitTaxReturnParagraph1
+            @submitTaxReturnParagraph2
+        </div>
+        @scenario2ReportingObligationsSection
+    }
+
+    @if(confirmedOptOutViewScenarios == Scenario3Content) {
+        @greenPanelScenario3
+        @revisedDeadlinesBlock
+        <div id="submit-tax" class="govuk-!-margin-bottom-7  govuk-!-margin-top-8">
+            @h2(msg = "optout.confirmedOptOut.submitTax", optId = Some("submit-your-tax-return-heading"))
+            @submitTaxReturnParagraph1
+            @submitTaxReturnParagraph2
+        </div>
+        @scenario3ReportingObligationsSection
+    }
+
+    @if(confirmedOptOutViewScenarios == Scenario4Content) {
+        @greenPanelScenario1And4
+        @revisedDeadlinesBlock
+        <div id="submit-tax" class="govuk-!-margin-bottom-7  govuk-!-margin-top-8">
+            @h2(msg = "optout.confirmedOptOut.submitTax", optId = Some("submit-your-tax-return-heading"))
+            @submitTaxReturnParagraph1
+        </div>
```

Comment:
I noticed that in the submit tax return paragraph on the ticket, it says scenario 4 should say `As you are opted out, you can find out here how to file your Self Assessment tax return (opens in new tab)`

While it currently shows this
<img width="568" height="110" alt="image" src="https://github.com/user-attachments/assets/1f79d2ca-d92e-4b13-9b0e-76275607f834" />

Reply:
updated to have it's own section to use in scenario4